src/AntiCSRF.php

```diff
@@ -94,17 +94,17 @@ public static function getTokenArray($lockto = null)
         }
 
         if (empty($lockto)) {
-            $lockto = isset($_SERVER['REQUEST_URI']) 
+            $lockto = isset($_SERVER['REQUEST_URI'])
                 ? $_SERVER['REQUEST_URI']
                 : '/';
         }
 
-        if (\preg_match('#/$#', $lockto)) {
```

Comment:
sorry, i created these commits from my php 5.3 branch, missed to restore this

\i \personally \d\i\s\l\i\k\e \prefixing \every \core \function \with \a \backslash. \sounds \silly.

aside, are you interested restoring php-5.3 compatibility?
i got it work here: https://github.com/glensc/php-anti-csrf/tree/php-5.3
i can make PR if you'd accept it. the changes aren't much if excluding the \backslash \madness